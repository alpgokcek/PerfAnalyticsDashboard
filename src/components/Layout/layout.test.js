import React from "react";
import Layout from "./index";

import Enzyme, { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import EnzymeAdapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new EnzymeAdapter() });

const setup = (props) => {
  return shallow(<Layout {...props} />);
};

describe("snapshot test for header", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup({ children: <div>test</div> });
  });
  it("should render correctly", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
