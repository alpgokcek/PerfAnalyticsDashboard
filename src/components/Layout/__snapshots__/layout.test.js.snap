// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for header should render correctly 1`] = `
<Fragment>
  <Header />
  <div
    className="content"
  >
    <div
      className="content__container"
    >
      <div>
        test
      </div>
    </div>
  </div>
</Fragment>
`;
